fiftyone/operators/delegated.py

```diff
@@ -80,7 +80,7 @@ def set_progress(self, doc_id, progress):
         """
         return self._repo.update_progress(_id=doc_id, progress=progress)
 
-    def set_running(self, doc_id, progress=None, run_link=None):
+    def set_running(self, doc_id, progress=None, run_link=None, required_state: ExecutionRunState = None):
```

Comment:
There was a recent argument about type hints. The resulting doctrine was that within a submodule, we have to keep it consistent. All type hints or no type hints.
Ritchie mentioned wanting to start with the `operators` submodule, but clearly this has not happened yet in `develop`. So best to remove type hints for your new parameter unless you want to take on the burden of adding them for the whole `operators` module.